### ISSUE TITLE
I need to confirm some questions to help me understand the code!

### ISSUE THREAD

**Issue by LokoBeta** [lengstrom/fast-style-transfer#77]
Hello,lengstrom! 

I meet the problem:
1/ The code of “fast-style-transfer” whether to follow the architecture "Perceptual Loss"?

2/ if 'YES',the code of 'transform.py' is a 'transform net'?

3/  if 'YES',why 'transform.net' only receive one parameter (is 'image')？why it have no the 'style model' ? Transform a style need a 'style model' 

3/ and, the code of 'vgg.py' is a 'loss net'?

thank you! 

I just started study ‘style transform’ Recently,  the answer is important for me! thank you!

**Comment by elleryqueenhomels** [comment]
@LokoBeta 
Q1: Yes
Q2: Yes. 'transform.py' is a 'Image Transformation Network' (short for 'ITN')
Q3: After learning, the weights in the ITN already 'contains' the style information. In fact, the working flow is: For a specific style, we use a bunch of content images to train the ITN. (1) we use an ITN to transform an input image x to y_hat, then using the style y_s, the content y_c (y_c == x) and the transformed image y_hat to pass through the pre-trained VGG-16 network to define the perceptual loss function. (2) After we get the loss function, we use Adam to minimize the loss function to update the weights of ITN. REMEMBER that the loss network (VGG-16) is fixed. Our goal is to update the ITN to minimize loss.
As a result, one style, one ITN.
